tests/hub_predtimechart/test_generate_target_data.py

```diff
@@ -49,3 +57,7 @@ def test_get_target_data_df():
     act_target_data_df = get_target_data_df(hub_dir, 'target-hospital-admissions-yes-na.csv')
     assert act_target_data_df["value"].dtype == pl.datatypes.Int64
     assert act_target_data_df["value"].to_list() == [3, 16, None, 106, 151, 23, 64, 8, 2, 266]
+
+    # case: file not found
```

Comment:
Related to earlier comment, it could be worth testing what happens if `None` is provided.